doc/cephadm/services/monitoring.rst

```diff
@@ -184,10 +184,79 @@ To see the default container images, run a command of the following form:
    DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.27.0'   
    DEFAULT_GRAFANA_IMAGE = 'quay.io/ceph/grafana:10.4.0'
 
+``cephadm`` stores a local copy of the ``cephadm`` binary in
+``var/lib/ceph/{FSID}/cephadm.{DIGEST}``, where ``{DIGEST}`` is an alphanumeric
+string representing the currently-running version of Ceph. In the Squid release
+of Ceph, this ``cephadm`` file is stored as a zip file and must be unzipped
+before its contents can be examined.
+
+This procedure explains how to generate a list of the default container images
+used by ``cephadm``.
+
+.. note:: This procedure applies only to the Sphinx release of Ceph. If you are
+   using a different version of Ceph, you cannot use this procedure to examine
```

Comment:
If we want to just have both versions in the docs, we should probably also add in the above section that that procedure only works with quincy and pacific

Reply:
No, I was implying the other part of this section right above these changes that has the quincy/pacific procedure for getting the default images 
![image](https://github.com/ceph/ceph/assets/47704447/2e2cb7e1-c2c5-4b22-9f80-762aed8511c2)

Reply:
...actually, three sets of docs, since __main__.py is associated with Reef but not with Squid.

In any case, this is a three-forked road I am willing to maintain.

Reply:
ah, okay, if you're going to remove in a follow up, then I'm fine with it.